tools/system_libs.py

```diff
@@ -530,6 +530,32 @@ def get_default_variation(cls, **kwargs):
     return super(MTLibrary, cls).get_default_variation(is_mt=shared.Settings.USE_PTHREADS, **kwargs)
 
 
+class OptimizedAggressivelyForSizeLibrary(Library):
+  def __init__(self, **kwargs):
+    self.is_optz = kwargs.pop('is_optz')
+    super(OptimizedAggressivelyForSizeLibrary, self).__init__(**kwargs)
+
+  def get_base_name(self):
+    name = super(OptimizedAggressivelyForSizeLibrary, self).get_base_name()
+    if self.is_optz:
+      name += '-optz'
```

Comment:
How about "-tiny" or "-small"?

Reply:
I don't think that is a better name, since it loses the meaning of what the corresponding `-O?` flag is. If I read `-tiny` or `-small`, I won't immediately know ("oh, do I get `-tiny` if I do `-Os`, or do I have to do pass `-Oz` for that?)

Reply:
OK that make sense.